hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/RootProxyWorldUpdater.java

```diff
@@ -96,6 +100,17 @@ public void commit() {
         final var contractChangeSummary = enhancement.operations().summarizeContractChanges();
         createdContractIds = contractChangeSummary.newContractIds();
 
+        context().ifPresent(context -> {
```

Comment:
There's a feature flag we should check here, `contracts.enforceCreationThrottle`, has a default value of `false`.